security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -38,6 +38,8 @@ The following advisory is available for the OpenShift Compliance Operator 1.4.0:
 [id="compliance-operator-1-4-0-bug-fixes"]
 === Bug fixes
 
+* Previously, on a cluster with Windows nodes, some rules will FAIL after auto remediation is applied because the Windows nodes were not skipped by the compliance scan. With this release, Windows nodes are correctly skipped when remediation is applied. (link:https://issues.redhat.com/browse/OCPBUGS-7355[*OCPBUGS-7355*])
```

Comment:
The Windows nodes should always be skipped by the Compliance Operator.
It's just that this behavior was made evident when remediation was applied and the rule still failed, because it was also scanning the Windows node.

I made a suggestion, but feel free to change and adapt.

```suggestion
* Previously, on a cluster with Windows nodes, some rules will FAIL after auto remediation is applied because the Windows nodes were not skipped by the compliance scan. With this release, Windows nodes are correctly skipped from scanning. (link:https://issues.redhat.com/browse/OCPBUGS-7355[*OCPBUGS-7355*])
```

Reply:
Thank you @yuumasato this has been done.